src/core/Alert/BaseAlert/BaseAlert.baseStyles.ts

```diff
@@ -0,0 +1,62 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseAlertBaseStyles = (theme: SuomifiTheme) => css`
+  &.fi-alert {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    width: 100%;
+
+    & .fi-alert_style-wrapper {
+      display: flex;
+      align-items: flex-start;
+    }
+
+    & .fi-alert_icon-wrapper {
+      margin-left: ${theme.spacing.m};
+      margin-top: ${theme.spacing.insetXl};
+
+      & .fi-alert_icon {
+        height: 24px;
+        width: 24px;
+      }
+    }
+
+    & .fi-alert_text-content-wrapper {
+      display: flex;
+      flex: 3 1 auto;
+      flex-direction: column;
+      padding: 0 20px;
+      margin-top: ${theme.spacing.s};
+      margin-bottom: ${theme.spacing.s};
+
+      & .fi-alert_content {
+        vertical-align: middle;
+        ${font(theme)('bodyTextSmall')}
+      }
+    }
+
+    /** Status variant styles */
+    &--neutral {
+      background-color: ${theme.colors.accentSecondaryLight1};
+      & .fi-alert_icon-wrapper .fi-icon .fi-icon-base-fill {
+        fill: #4698c3; /* Later add theme.colors.example styling */
+      }
+    }
+
+    &--error {
+      background-color: ${theme.colors.alertLight1};
+      & .fi-alert_icon-wrapper .fi-icon .fi-icon-base-fill {
+        fill: ${theme.colors.alertBase};
+      }
+    }
+
+    &--warning {
+      background-color: #fff6e0; /** needs to be warningLight1 but the token is yet to be added */
+      & .fi-alert_icon-wrapper .fi-icon .fi-icon-base-fill {
+        fill: #b54439; /* Later add theme.colors.example styling */
```

Comment:
Maybe change little more describing comment because 
`/* Later add theme.colors.example styling */` is confusing as we might not want to add stuff named like that.

If existing token in our code bases does not match with the design (design have e.g updated the tokens in design), we should tell that instead.


**Check and fix  also other baseStyles of Alert/InlineAlert for this kind of comments.**